src/Data/Misc.lua

```diff
@@ -5,8 +5,11 @@ local data = ...
+data.minionDamageTable = { 5.6199998855591, 6.0300002098083, 6.460000038147, 6.9200000762939, 7.4099998474121, 7.9299998283386, 8.4799995422363, 9.0600004196167, 9.6800003051758, 10.329999923706, 11.029999732971, 11.760000228882, 12.539999961853, 13.369999885559, 14.25, 15.170000076294, 16.159999847412, 17.200000762939, 18.299999237061, 19.459999084473, 20.700000762939, 22, 23.389999389648, 24.85000038147, 26.389999389648, 28.030000686646, 29.760000228882, 31.579999923706, 33.520000457764, 35.560001373291, 37.720001220703, 40, 42.409999847412, 44.959999084473, 47.639999389648, 50.490001678467, 53.490001678467, 56.659999847412, 60, 63.529998779297, 67.26000213623, 71.199996948242, 75.360000610352, 79.73999786377, 84.370002746582, 89.25, 94.400001525879, 99.839996337891, 105.56999969482, 111.62000274658, 118, 124.73000335693, 131.83000183105, 139.30999755859, 147.19999694824, 155.52000427246, 164.2799987793, 173.5299987793, 183.27000427246, 193.53999328613, 204.36999511719, 215.7799987793, 227.80000305176, 240.46000671387, 253.80999755859, 267.86999511719, 282.69000244141, 298.29000854492, 314.73001098633, 332.04998779297, 350.29000854492, 369.5, 389.73001098633, 411.04000854492, 433.4700012207, 457.08999633789, 481.9700012207, 508.14999389648, 535.71997070312, 564.75, 595.29998779297, 627.46002197266, 661.30999755859, 696.95001220703, 734.45001220703, 773.90997314453, 815.45001220703, 859.15997314453, 905.15002441406, 953.53997802734, 1004.4699707031, 1058.0400390625, 1114.4100341797, 1173.7099609375, 1236.0999755859, 1301.7299804688, 1370.7600097656, 1443.3800048828, 1519.7600097656, 1600.0899658203, }
```

Comment:
For consistency maybe we should name this `monsterAllyDamageTable`, or rename `monsterAllyLifeTable` to `minionLifeTable`